src/exchanges/bitfinex-client.js

```diff
@@ -9,7 +9,18 @@ const Level3Snapshot = require("../level3-snapshot");
 const Level3Update = require("../level3-update");
 
 class BitfinexClient extends BasicClient {
-  constructor({ wssPath = "wss://api.bitfinex.com/ws/2", watcherMs, l2UpdateDepth = 250 } = {}) {
+  /**
+   *
+   * @param {Object} params
+   * @param {Boolean} [params.enableEmptyHeartbeatEvents]       (optional, default false). if true, emits empty events for all channels on heartbeat events which includes the sequenceId.
```

Comment:
new constructor param `enableEmptyHeartbeatEvents`, to enable empty `Ticker` and `Trade` events w/the `sequenceId` when these channels receive a heartbeat